commit_gallery_1

PopovaAnna1209 opened the pull request:
Фух)))первый пошел)))

PopovaAnna1209 committed:
commit_gallery_1

RomarioNik commented on src/sass/components/_section-gallery.scss:
```diff
@@ -0,0 +1,93 @@
+.gallery {
+
+
+background-image: linear-gradient(( #84A178 25%, transparent 25%), (#84A178 25%, transparent 25%));
```
#84A178
есть переменная для цвета фона. Посмотри везде по коду и замени, пожалуйста
--bg-color: #84a178;

RomarioNik commented on src/partials/section-gallery.html:
```diff
@@ -1,3 +1,46 @@
+        <img src="/src/images/images-gallery/ice_des2@1x.jpg" alt="Ice2" width="1013">
+      </picture>
+    </div> 
+  </div>
```
или img или обертке нужно задать border-radius: 24px;
Видны белые углы когда картинка попадает на зеленый фон

PopovaAnna1209 committed:
commit_gallery_2

PopovaAnna1209 committed:
commit_gallery_3

1. в анимации поменяла ключевые точки, но у меня все равно первую анимацию рвет, а потом так как должно работает
2. падинги убрала
3. вендорные префиксы убрала

PopovaAnna1209 committed:
commit_gallery_4

немного код подправила

PopovaAnna1209 committed:
commit_gallery_5

с правильными изображениями

Commit:
some changes
